src/main/c/ssl.c

```diff
@@ -1615,6 +1615,12 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
         return 0;
     }
 
+    if (ssl_->session == NULL) {
```

Comment:
Maybe we should file a bug against BoringSSL?

https://www.openssl.org/docs/manmaster/ssl/SSL_SESSION_get_time.html
> If any of the function is passed the NULL pointer for the session s, 0 is returned.